cargo-shuttle/src/lib.rs

```diff
@@ -210,6 +215,29 @@ impl Shuttle {
         .context("failed to deploy cargo project")
     }
 
+    async fn check_lib_version(&self, project_args: ProjectArgs) -> Result<()> {
+        let cargo_path = project_args.working_directory.join("Cargo.toml");
+        let cargo_doc = read_to_string(cargo_path.clone())?.parse::<Document>()?;
+        let current_shuttle_version = &cargo_doc["dependencies"]["shuttle-service"]["version"];
+        let service_semver = Version::parse(current_shuttle_version.as_str().unwrap())?;
+        let mut server_version = String::new();
+
+        client::shuttle_version(self.ctx.api_url())
+            .await
+            .map(|response| server_version = response)?;
```

Comment:
```suggestion
        let server_version =  client::shuttle_version(self.ctx.api_url()).await?;
```
Hopefully I understand this correctly :see_no_evil:

Reply:
thanks!